tests/tests.toml

```diff
@@ -429,6 +429,14 @@ cpp = true
 filename = "../integration_tests/test_bool_binop.py"
 asr = true
 
+[[test]]
+filename = "../integration_tests/test_logical_compare.py"
+asr = true
+
+[[test]]
+filename = "../integration_tests/test_logical_assignment.py"
+asr = true
+
```

Comment:
Please remove these from here.

Reply:
>Same must be done for the error tests

We need one error reference test to check for semantic error when different types are used with the `selective or` operation.

Reply:
I am sorry for asking the same question, my question stems from the fact that we have many integration_tests listed in the above file. This is a bit confusing.